src/http/structs.rs

```diff
@@ -43,6 +48,7 @@ pub struct AybConfig {
     pub email: AybConfigEmail,
     pub web: Option<AybConfigWeb>,
     pub cors: AybConfigCors,
+    pub isolation: Option<AybConfigIsolation>,
 }
 
 impl AybConfig {
```

Comment:
tiny hack: I've checked in a built `nsjail` binary to `tests/nsjail`. In a perfect world I'd containerize it or build it each time from scratch, both of which feel like a decent amount of work given it's just for tests and not actually distributing the binary.

Reply:
Good push --- it was more laziness for not wanting to update GitHub actions to build `nsjail` :P.

We now have
* just the script
* GH actions builds `nsjail`
* documentation in README.md on isolation (yay!) and how to run tests (because you now have to build `nsjail` once, yay!)